static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -110,61 +99,59 @@ define([
             usMessagingTestVariant = ab.testCanBeRun('MembershipEngagementUsMessageCopyExperiment') ? ab.getTestVariantId('MembershipEngagementUsMessageCopyExperiment') : undefined,
             linkHref = formatEndpointUrl(edition, message);
 
-        if (messagingTestVariant) {
-            var messagingTestName = 'messaging-test-1';
-            if (messagingTestVariant !== 'notintest') {
-                var variantMessages = {
-                    fairer: 'We all want to make the world a fairer place. We believe journalism can help – but producing it is expensive. That\'s why we need Supporters. Join for £49 per year.',
-                    appreciate: 'Become a Supporter and appreciate every article, knowing you\'ve helped bring it to the page. Be part of the Guardian. Join for £49 per year.',
-                    secure: 'Secure the future of independent journalism. Help us create a fairer world. Support the Guardian for £49 per year.'
-                };
-                campaignCode = 'gdnwb_copts_mem_banner_messaging1uk' + '__' + messagingTestVariant;
-                linkHref = endpoints[edition] + '?INTCMP=' + campaignCode;
-                customOpts = {
-                    testName: messagingTestName,
-                    setEngagementText: messagingTestVariant === 'control' ? undefined : variantMessages[messagingTestVariant]
-                };
-                customJs = getCustomJs;
-            }
+        if (messagingTestVariant && messagingTestVariant !== 'notintest') {
+            var variantMessages = {
+                Get_round_to: 'Not got round to supporting us yet? Now is the time. Give £5 a month today',
+                Give_upfront: 'Give £5 a month to help support our journalism and make the Guardian\'s future more secure',
+                Together_informed: 'Together we can keep the world informed. Give £5 a month to support our journalism',
+                Coffee_5: 'For less than the price of a coffee a week, you could help secure the Guardian’s future. Support our journalism for £5 a month'
```

Comment:
the apostrophe used here is different to the single quote used in `Give_upfront`